tensorflow/core/protobuf/config.proto

```diff
@@ -443,6 +444,27 @@ message ConfigProto {
     // If true, use NCCL for CollectiveOps.  This feature is highly
     // experimental.
     bool collective_nccl = 7;
+    
+    // In the following words, session state means the value of a variable,
```

Comment:
@josh11b Are the following description words clear for uses to understand?
Appreciate your review. Thanks!

Reply:
I'd delete the word "words"

Reply:
@josh11b Done :)
The "words" is deleted in the new commit. Thx!